ethereum_history_api/oracles/src/noir/oracles/common/encode.ts

```diff
@@ -45,6 +45,16 @@ export function encodeHex(hexString: string): Hex[] {
   return chunks;
 }
 
+export function encodeArray(array: Uint8Array, padLength: number) {
+  return padArray(
+    Array.from(array)
+      .map((it) => it.toString(16))
+      .map((it) => `0x${it.length < BYTE_HEX_LENGTH ? '0' : ''}${it}`),
+    padLength,
+    ZERO_PAD_VALUE
+  );
+}
+
```

Comment:
Please use this as a reference: https://github.com/vlayer-xyz/noir-ethereum-history-api/pull/158

Reply:
I'm not sure how to apply #158 here. If you mean to do a similar thing to encodeProofNode, I don't think I can do that here since I am encoding an UInt8Array instead of a hex string